app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -48,105 +49,81 @@
+        android:layout_height="match_parent"
+        android:scrollbars="none">
 
         <androidx.constraintlayout.widget.ConstraintLayout
           android:layout_width="match_parent"
           android:layout_height="wrap_content">
 
-          <TextView
-            android:id="@+id/heading_text"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="40dp"
-            android:fontFamily="sans-serif-medium"
-            android:lines="2"
-            android:paddingStart="28dp"
-            android:paddingEnd="28dp"
-            android:text="@string/admin_auth_heading"
-            android:textColor="@color/oppiaPrimaryText"
-            android:textSize="20sp"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+          <Button
+            android:id="@+id/submit_button"
+            style="@style/StateButtonActive"
+            android:layout_marginTop="@dimen/submit_margin_top_bottom"
+            android:layout_marginBottom="@dimen/submit_margin_top_bottom"
+            android:background="@{viewModel.isSubmitButtonActive ? @drawable/state_button_primary_background :@drawable/start_button_transparent_background}"
+            android:minHeight="@dimen/min_height"
+            android:text="@string/state_continue_submit"
+            android:textColor="@{viewModel.isSubmitButtonActive ? @color/white : @color/submitButtonInactiveText }"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="@+id/input_pin"
+            app:layout_constraintHorizontal_bias="1.0"
+            app:layout_constraintStart_toStartOf="@+id/input_pin"
+            app:layout_constraintTop_toBottomOf="@id/input_pin" />
 
           <TextView
             android:id="@+id/sub_text"
-            android:layout_width="match_parent"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginTop="12dp"
+            android:layout_marginTop="@dimen/subtitle_margin_top"
             android:fontFamily="sans-serif"
-            android:paddingStart="28dp"
-            android:paddingEnd="28dp"
             android:text="@string/admin_auth_sub"
             android:textColor="@color/black_54"
-            android:textSize="16sp"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
+            android:textSize="@dimen/subtitle_text"
+            app:layout_constraintEnd_toEndOf="@+id/heading_text"
+            app:layout_constraintStart_toStartOf="@+id/heading_text"
             app:layout_constraintTop_toBottomOf="@id/heading_text" />
 
-          <LinearLayout
-            android:id="@+id/admin_auth_linearLayout"
-            android:layout_width="match_parent"
+          <TextView
+            android:id="@+id/heading_text"
```

Comment:
You can follow this convention `admin_auth_heading_textview` for ids to make it more specific and make sure you also change it in your testcases if its getting used.

Reply:
ok. Updating `heading_text `, `sub_text` and `input_pin` with the `admin_auth` prefix, this way making it more specific to the related file.

Reply:
Yes thanks.